eiffellib/events/eiffel_base_event.py

```diff
@@ -174,7 +174,7 @@ class EiffelBaseEvent(object):
     tag = "_"
     domain_id = "_"
     version = "0.0.1"
-    meta = EiffelBaseMeta("EiffelBaseEvent", version)
+    meta = EiffelBaseMeta(__qualname__, version)
```

Comment:
`__qualname__` will break our python2 compatibility, which can be considered fine if we do a major tag update.
But we will have to discuss whether or not we are okay with this first.

Reply:
I'd be fine with doing a major version step and effectively leaving python 2.x behind, although I think we need to hold of and provide any potential users with a deprecation notice/period. What I'm reluctant to do is to maintain both a python 2.x and a python 3.x versions, but that is mostly common sense given the EOL of Python 2.x.